app/code/Bulbulatory/Recommendations/Model/Recommendation.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Bulbulatory\Recommendations\Model;
+
+use Magento\Framework\DataObject\IdentityInterface;
+use Magento\Framework\Model\AbstractModel;
+
+/**
+ * Class Recommendation
+ * @package Bulbulatory\Recommendations\Model
+ */
+class Recommendation extends AbstractModel implements IdentityInterface
+{
+    protected $_cacheTag = 'bulbulatory_recommendations_recommendation';
+    protected $_eventPrefix = 'bulbulatory_recommendations_recommendation';
+
+
+    protected function _construct()
+    {
+        $this->_init('Bulbulatory\Recommendations\Model\ResourceModel\Recommendation');
```

Comment:
Gdy potrzebna jest nazwa klasy, lepiej użyć ::class niż podać nazwę klasy jako string.
```suggestion
        $this->_init(Bulbulatory\Recommendations\Model\ResourceModel\Recommendation::class);
```
PHP sam sobie to zamieni na klasę jako string, ale sama praca jest wygodniejsza, ponieważ w związku z tym, że nie operujemy na stringach, wiele IDE będzie oferowało np. inne formatowanie takiego kawałka kodu albo w przypadku zmiany nazwy klasy automatycznie podmieni wszystkie jej wystąpienia. Jeśli będziemy chcieli sprawdzić, co to za klasa, to w wielu edytorach także będzie można bezpośrednio do niej przejść z tego miejsca, a nie ręcznie wyszukiwać.